Move to Linux Foundation Charities 501(c)(3) and establish the Jupyter Foundation directed fund as part of Linux Foundation 501(c)(6) (#226)
jupyter/governance: jupyter_foundation.md

# Jupyter Foundation

> The Jupyter Foundation is a directed fund of the Linux Foundation 501(c)(6). It exists to provide resources and strategic counsel to Project Jupyter. The charter is here (TODO: Link). We summarize relevant points from the charter below.

Reviewer: FYI, we still need to link to a draft of the charter (which we are close to having a draft of, and will be available for review before we vote). Also, we should clarify here that the charter is the source of truth for this information, and any discrepancy between this document and the charter should defer to the charter.

Reviewer: Ana and I pushed a few changes, including adding a sentence around this.

> ## Jupyter Foundation Governing Board

Reviewer 2: I unfortunately missed the last EC meetings due to travel/illness, so forgive me if this was discussed to death already. But my inclination would be to name this "something something Council" instead of "Governing Board" to keep in line with our standard naming practices for most of our decision making bodies of this type.

If there's a good reason to divert from our practice, we should encode that in the [FAQ](https://jupyter.org/governance/linux-proposal.html) (right now I see no mention of the word "Board" there).

Reviewer 2: Just discussed it, keeping it as "Jupyter Foundation Governing Board" b/c:

1. That's the standard terminology already used by LF in many other places, and it will benefit us greatly when fundraising for this specific body to use established patterns.
2. This is really a new type of group, whose job is to govern our new fund under the 501(c)6. We've never had anything quite like that, so it is actually useful to signal its difference via naming.

This makes sense to me, resolving.

The Jupyter Foundation Governing Board controls the resources of the Jupyter Foundation and works to build a healthy collaboration between the Jupyter community, Jupyter leadership, and members of the Jupyter Foundation, including for-profit organizations. The current members of the Jupyter Foundation Governing Board are listed in the [Leadership Directory](people.md#jupyter-foundation-governing-board).

## Responsibilities of the Jupyter Foundation Governing Board

> - Serve Project Jupyter and its mission.

Reviewer: To copy language from the draft Jupyter Foundation charter, let's change "Serve" to "Support"
```suggestion
- Support Project Jupyter and its mission.
```

Author: :+1:

- Meet regularly to review strategic objectives of Jupyter.
- Develop and approve an annual strategic plan and budget for funding from the Jupyter Foundation to benefit Project Jupyter.

> - Recruit and retain Jupyter Foundation members.

Reviewer 2: Not sure about the ideal wording, but I think we should explicitly list budgetary decisions/allocations on other potential timescales. I don't imagine that a single annual plan/budget would be the only mechanism for allocating funds we should have.

Perhaps something like 

> "Meet at least quarterly and allocate budgetary resources from the fund as appropriate, based on both the annual strategic plan and potential needs and opportunities that may arise and be brought up to the Board for consideration". 

Probably too wordy, but it's a start...

## Membership

The Jupyter Foundation Governing Board consists of:

> - All members of the [Jupyter Executive Council](./executive_council.md)

Reviewer 2: Missing periods at the end of these items.

Author: Fixed.

- One representative from each Jupyter Foundation Premier Member

> - No more than two representatives total from all Jupyter Foundation General Members

Reviewer: The current draft of the charter has:
```suggestion
- Between one and three representatives total from all Jupyter Foundation General Members
```

## Decision making

> The Jupyter Foundation Governing Board follows the decision-making policies in its charter. (TODO: Link)

Reviewer 3: To resolve any ambiguities here between our decision making policies and those of the LF, in addition to providing a link, we should clarify that this is the "Jupyter Decision Making policy".

Reviewer 2: Other than adding the links (coming soon) we think this is clear enough - its charter spells out its specific decision making process, which is fairly straightforward but has more formal voting requirements in order to keep a clear record. I think it's unambiguous and we don't need to repeat it here, people can read the doc.

Reviewer: I've clarified in a new commit here that the Jupyter Foundation Governing Board follows the decision-making guidelines in the Jupyter Foundation charter.
